src/homedelivery/query.js

```diff
@@ -35,7 +35,9 @@ async function queryZuora (deliveryDate, config: Config) {
      ProductRatePlanCharge.ProductType__c = 'Print ${deliveryDay}' AND
      Product.Name = 'Newspaper Delivery' AND
      RatePlanCharge.EffectiveStartDate <= '${formattedDate}' AND
-     RatePlanCharge.EffectiveEndDate >= '${formattedDate}' AND
+     (Subscription.Status = 'Active' AND RatePlanCharge.EffectiveEndDate >= '${formattedDate}' OR
+      Subscription.Status = 'Cancelled' AND RatePlanCharge.EffectiveEndDate > '${formattedDate}'
+     ) AND
```

Comment:
with this new condition I think technically we do not need the first condition (Subscription.Status = 'Active' OR Subscription.Status = 'Cancelled') but I left it there for clarity

Reply:
This sounds pretty reasonable, I also did this on the upcoming PR to do the same to weekly.